UPGRADING.md

```diff
@@ -28,5 +31,32 @@ max_num_outbound_peers = 10
 ```
 
 As you can see, the default ratio of inbound/outbound peers is 4/1. The reason
-as we want it to be easier for new nodes to connect to the network. You can
+is we want it to be easier for new nodes to connect to the network. You can
 tweak these parameters to alter the network topology.
+
+### RPC Changes
+
+The result of `/commit` used to contain `header` and `commit` fields at the top level. These are now contained under the `signed_header` field.
+
+### ABCI Changes
+
+The header has been upgraded and contains new fields, but none of the existing
+fields were changed, except their order.
+
+The `Validator` type was split into two, one containing an `Address` and one
+containing a `PubKey`. When processing `RequestBeginBlock`, use the `Validator`
+type, which contains just the `Address`. When returning `ResponseEndBlock`, use
+the `ValidatorUpdate` type, which contains just the `PubKey`.
+
+### Validator Set Updates
+
+Validator set updates returned in ResponseEndBlock for height `H` used to take
```

Comment:
Don't think we should duplicate changelog entries here (except when they lead to clear actions like change this or do that)

Reply:
Ok - wasn't sure how to think about this. The point is that for val sets they do have to change the way they track them in the app and I wanted to better indicate that. I'll add more text.